test/e2e/parallel/spacebindingrequest_test.go

```diff
@@ -0,0 +1,267 @@
+package parallel
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/gofrs/uuid"
+
+	spacebindingrequesttestcommon "github.com/codeready-toolchain/toolchain-common/pkg/test/spacebindingrequest"
+
+	testspace "github.com/codeready-toolchain/toolchain-common/pkg/test/space"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestCreateSpaceBindingRequest(t *testing.T) {
+	// given
+	t.Parallel()
+	// make sure everything is ready before running the actual tests
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member1()
+
+	t.Run("success", func(t *testing.T) {
+		t.Run("create space binding request", func(t *testing.T) {
+			// when
+			// we create a space to share , a new MUR and a SpaceBindingRequest
+			space, spaceBindingRequest, spaceBinding := NewSpaceBindingRequest(t, awaitilities, memberAwait, hostAwait, "admin")
```

Comment:
I named it this way because we also have `NewSpaceRequest` , which does similar thing but for `SpaceRequest` , if we want to rename it I would opt for  `EnsureSpaceBindingRequest`.

Reply:
Another alternative could be: `setupSpaceBindingRequest` .

Reply:
what about `CreateSpaceBindingRequest`, since we also have `CreateSpace`? (which is also called in this func, btw)

Reply:
`CreateSpaceBindingRequest` makes sense , but I've already used it here https://github.com/codeready-toolchain/toolchain-e2e/pull/779/files#diff-d21939085e6a53e4b5e5b3b1158e837fe749eee6e124ba8a29645c714f0a859dR33 😄  , so not sure ...

Reply:
naming is hard :smile: